shared/macros.jinja

```diff
@@ -3,6 +3,12 @@ ocil_clause: "the required value is not set"
 {{%- endmacro %}}
 
 
+{{% set dconf_gdm_dir = "distro.d" %}}
+{{% if product not in ("fedora", "rhel9") %}}
+{{% set dconf_gdm_dir = "gdm.d" %}}
```

Comment:
I have a question: Shouldn't it be a property of product.yml? What's our guidance?

Reply:
That would probably be a better place, yes. Taking a closer look, it looks like that the issue is present only in RHEL7 and RHEL8, so a default value combined with override should work better.

Reply:
Something similar was done recently for grub2's path:
https://github.com/ComplianceAsCode/content/pull/6986

Reply:
> Something similar was done recently for grub2's path

Thanks, it is nice to have a provisional solution that is already used.